core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -54,17 +58,29 @@
    * Factory for {@link FileSystem}.
    */
   class Factory {
+    private static final Logger LOG = LoggerFactory.getLogger(Factory.class);
 
     private Factory() {} // prevent instantiation
 
     public static FileSystem get() {
-      if (Configuration.getBoolean(PropertyKey.USER_LINEAGE_ENABLED)) {
-        return LineageFileSystem.get(FileSystemContext.INSTANCE, LineageContext.INSTANCE);
-      }
-      return BaseFileSystem.get(FileSystemContext.INSTANCE);
+      return get(FileSystemContext.INSTANCE);
     }
 
     public static FileSystem get(FileSystemContext context) {
+      if (LOG.isDebugEnabled()) {
+        TreeMap<String, String> keyValueSet = new TreeMap<>(Configuration.toMap());
```

Comment:
nit: declare as `Map`
Why do we need to copy to a TreeMap?

Reply:
to sort the properties. Otherwise `HashMap` will make the output in hash order.

Reply:
ah that makes sense. Perhaps add a comment that we're doing it to sort by key?